### DISCUSSION TITLE
請問在星陳輸入法中，“彡”的字根編碼是cs還是cp？

### DISCUSSION THREAD

**Discussion by 11201010** [forFudan/yuhao#45] [Q&A]
因為“彡”在[星陳的字根圖](https://yuhao.forfudan.com/yustar.png)顯示為Cs，但[​“彡”的星陳拆分查询](https://yuhao.forfudan.com/zht/chaifen/?q=彡)顯示為Cpp。
兩者的字根編碼不一致，所以想確認在[字根編碼​中前 51 到 100 的常用字根](https://yuhao.forfudan.com/learn/roots.html#常用字根)的“彡”的字根編碼應該為何？謝謝。

**Comment by 11201010** [comment]
我把星陳字根圖的所有字根都嘗試輸入一遍，發現其他字根也有字根圖與拆分查詢不一致的問題。
不一致的星陳字根系列如下：
星陳字根系列|字根圖|拆分查詢
-|-|-
彡|cs|cp
小|ix|iv
七|xq|xi

因此，想請問字根的編碼應該以字根圖還是以拆分查詢為主？謝謝。

**Reply by forfudan** [reply]
你好，是字根图有问题。谢谢你帮忙找出来这几个错误！（我一个人审核眼睛都瞎了
